src/imp/tracer_imp.js

```diff
@@ -883,11 +883,13 @@ export default class TracerImp extends EventEmitter {
         let clockOffsetMicros = this._clockState.offsetMicros();
 
         // Diagnostic information on the clock correction
-        this.logStable('cr/time_correction_state', {
-            offset_micros  : clockOffsetMicros,
-            active_samples : this._clockState.activeSampleCount(),
-            ready          : clockReady,
-        });
+        if (this._options.debug) {
```

Comment:
This is the right idea, but this likely should use the internal logging method instead of checking `debug` (which I'm trying to remove in favor of `verbose`). This is likely what you want, as it does the verbosity check already:

```js
this._infoV(1, 'time correction state', { 
   ...
});
```

https://github.com/lightstep/lightstep-tracer-javascript/blob/master/src/imp/tracer_imp.js#L1013